src/style.css

```
@@ -0,0 +1,69 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  --color-blue: #00acdc;
+  --header-height: 60px;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
+  width: calc(var(--header-height) - 20px);
+  height: calc(var(--header-height) - 20px);
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.nav__list {
+  position: relative;
+  display: flex;
+
+  margin: 0;
+  padding: 0;
+  margin-right: 50px;
+  list-style: none;
+}
+
+.nav__item {
+  margin-right: 20px;
+  flex-basis: auto;
+  flex-shrink: 0;
+}
+
+.nav__item:last-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: var(--header-height);
+
+  text-decoration: none;
+  color: #000;
+}
+
+.is-active,
+.nav__link:hover {
+  color: var(--color-blue);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  bottom: 0;
+  width: 37px;
```

```suggestion
  width: 100%;
```
in that way it will work for every link does not matter what length it would be